conans/test/functional/layout/test_editable_cmake_components.py

```diff
@@ -32,6 +32,7 @@ def build(self):
 
             def layout(self):
                 cmake_layout(self, src_folder="src")
+                self.cpp.source.includedirs = []
```

Comment:
It is important to invalidate the non-existing ``includedirs`` in ``conanfile.cpp.source.includedirs = ["include"]`` inside ``cmake_layout()``